include/z64player.h

```diff
@@ -188,9 +198,93 @@ typedef enum {
     /* 34 */ PLAYER_MWA_MAX
 } PlayerMeleeWeaponAnimation;
 
-typedef struct {
-    /* 0x00 */ f32 unk_00;
-    /* 0x04 */ f32 unk_04;
+typedef enum PlayerDoorType {
+    /* -1 */ PLAYER_DOORTYPE_TALK = -1, // Prevents opening the door normally
```

Comment:
```suggestion
typedef enum PlayerDoorType {
    /* -1 */ PLAYER_DOORTYPE_TALKING = -1, // Displays a message instead of opening
```
"talking" parallels "sliding" more, I suppose. And reworded the comment a bit. dont think "normally" is needed, since even in the special case of the milk bar -1 is still used to talk, then its changed to the proximity type later

Reply:
oh i understand, "normally" meaning "opening a door noramlly". Not "normally, this makes the door talk".
Eitherway i still think phrasing it as "talk instead of open" is an improvement